perfscale_regression_ci/scripts/custom_workload_env.sh

```diff
@@ -6,5 +6,5 @@ export WORKLOAD=${WORKLOAD:-"custom"}
 export INDEXING=${INDEXING:-"false"}
 export METRICS_PROFILE=${METRICS_PROFILE:-"metrics-profiles/metrics.yaml"}
 export ALERTS_PROFILE=${ALERTS_PROFILE:-""}
-export COMPARISON_CONFIG="clusterVersion.json nodeMasters-max.json nodeAggWorkers.json"
-export GEN_CSV=true
\ No newline at end of file
+#export COMPARISON_CONFIG="clusterVersion.json nodeMasters-max.json nodeAggWorkers.json"
```

Comment:
I think instead of commenting these out we should just have clusterversion as the default for comparison config and gen_csv false and other scripts can reset and generate csv as they want 

@svetsa-rh we found you need to update your scripts to set the metrics profile to metrics-aggregated correct? Can we open an issue  to make sure we update that properly

Reply:
@paigerube14 Updated comparison config to have clusterversion and gen_csv to false.

Reply:
@paigerube14 Sure, will do that.